Log/lambda_function.py

```diff
@@ -30,6 +30,11 @@
     "ddsourcecategory": "aws",
 }
 
+try:
```

Comment:
@aeikenberry thinking of having Terraform map variable impact the default metadata on each of these logs...is this the right python code to support that?